packages/bpk-mixins/src/mixins/_buttons.scss

```diff
@@ -30,27 +30,6 @@
 ///     @include bpk-button();
 ///   }
 
-$vars: (
```

Comment:
Please don't remove this I still need it

Reply:
moved it to tokens/base.default.scss
is that okay?

Reply:
Don't put it in that file, it gets autogenerated and will be overwitten

Reply:
Moved it to a separate file - `_theming.scss` that's included in `_index.scss`.